build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.10'
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.16'
```

Comment:
I wonder is it possible to apply different versions on the condition of Gradle version? @msgilligan

Reply:
I might be able to do that...

Reply:
@schildbach See: https://github.com/bitcoinj/bitcoinj/pull/2140